backend/pkg/engine/processor.go

```diff
@@ -361,28 +363,24 @@ func newEngine(
 
 func NewProcessor(
 	params *models.Parameters,
+	cluster cluster.Cluster,
 	kyvernoConfig *corev1.ConfigMap,
 	dClient dclient.Interface,
-	factory engineapi.ContextLoaderFactory,
+	cmResolver engineapi.ConfigmapResolver,
```

Comment:
hm not sure if its needed to pass cmResolver only to create a factory out of it. Why not passing directly the factory?

Reply:
we probably don't want the caller to create the factory, the engine should be autonomous IMO.
it makes the caller life more difficult than necessary.